benchmark/Manifest.toml

```diff
@@ -0,0 +1,488 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
This file should be excluded from the PR; it is not needed.

Reply:
I have been following  [BenchmarkCIExample.jl](https://github.com/tkf/BenchmarkCIExample.jl) there they have  kept Manifest.toml in the benchmarking folder.
But now I see the BenchmarkCI.jl working without Manifest.toml, looks like we don't need it.